### PR TITLE
feature: Add Postgres application support

### PR DESCRIPTION
**Pull request by VVelox**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [X] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [X] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Adds support for polling Postgres.

This requires https://github.com/librenms/librenms-agent/pull/91 .

### CODE REVIEW

**Comment by VVelox** [comment]
Unneeded require lines in graphs removed and tested.

File: doc/Extensions/Applications.md [hunk 1]
```diff
@@ -518,3 +519,27 @@ extend fail2ban /etc/snmp/fail2ban
...
+extend postgres /etc/snmp/postgres
+```
+
+4: Restart snmpd on your host
+
+5: Install the Nagios check check_postgres.pl on your system.
+
+6: Verify the path to check_postgres.pl in /etc/snmp/postgres is correct.
+
+7: If you wish it to ignore the database postres for totalling up the stats, set ignorePG to 1(the default) in /etc/snmp/postgres. If you are using netdata or the like, you may wish to set this or otherwise that total will be very skewed on systems with light or moderate usage.
```
Comment (laf) [inline doc/Extensions/Applications.md]: `database postres` - spelling mistake?

File: includes/polling/applications/postgres.inc.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+use LibreNMS\RRD\RrdDefinition;
+
+$options      = '-O qv';
+$mib          = 'NET-SNMP-EXTEND-MIB';
+$oid          = 'nsExtendOutputFull.8.112.111.115.116.103.114.101.115';
+$postgres = snmp_walk($device, $oid, $options, $mib);
+
+list($backends, $commits, $rollbacks, $read, $hit, $idxscan, $idxtupread, $idxtupfetch, $idxblksread,
+    $idxblkshit, $seqscan, $seqtupread, $ret, $fetch, $ins, $upd, $del ) = explode("\n", $postgres);
```
Comment (laf) [inline includes/polling/applications/postgres.inc.php]: Extra space after `$del`

**Comment by laf** [comment]
Oh and a rebase.

File: doc/Extensions/Applications.md (hunk 1 above)
Comment (VVelox) [inline doc/Extensions/Applications.md]: Yup. That it is.

**Comment by VVelox** [comment]
Changes made now. :)

File: doc/Extensions/Applications.md [hunk 3]
```diff
@@ -539,7 +539,7 @@ extend postgres /etc/snmp/postgres
 
 6: Verify the path to check_postgres.pl in /etc/snmp/postgres is correct.
 
-7: If you wish it to ignore the database postres for totalling up the stats, set ignorePG to 1(the default) in /etc/snmp/postgres. If you are using netdata or the like, you may wish to set this or otherwise that total will be very skewed on systems with light or moderate usage.
+7: If you wish it to ignore the database postges for totalling up the stats, set ignorePG to 1(the default) in /etc/snmp/postgres. If you are using netdata or the like, you may wish to set this or otherwise that total will be very skewed on systems with light or moderate usage.
```
Comment (laf) [inline doc/Extensions/Applications.md]: :) One spelling mistake to another :D

Comment (VVelox) [inline doc/Extensions/Applications.md]: Hah! Was just about to reply with saying it is suppose to be lower case when I noticed the missing r.

**Pull request merged**